l1-contracts/contracts/dev-contracts/test/DummySharedBridge.sol

```diff
@@ -2,14 +2,14 @@
 
 pragma solidity 0.8.24;
 
-import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IERC20} from "@openzeppelin/contracts-v4/token/ERC20/IERC20.sol";
 
 import {L2TransactionRequestTwoBridgesInner} from "../../bridgehub/IBridgehub.sol";
 import {TWO_BRIDGES_MAGIC_VALUE, ETH_TOKEN_ADDRESS} from "../../common/Config.sol";
-import {PausableUpgradeable} from "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
+import {PausableUpgradeable} from "@openzeppelin/contracts-upgradeable-v4/security/PausableUpgradeable.sol";
 import {Ownable2StepUpgradeable} from "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
 import {UnsafeBytes} from "contracts/common/libraries/UnsafeBytes.sol";
```

Comment:
```suggestion
import {UnsafeBytes} from "../../common/libraries/UnsafeBytes.sol";
```

Reply:
Relative import is used everywhere else. I know it is not introduced with this PR, but please include this change too :)

Reply:
i dont believe its used we can probably just remove it